README.md

```diff
@@ -90,6 +91,33 @@ sensu_asset { 'test':
 }
 ```
 
```

Comment:
should update the `Compatibility` section with a pointer to sensu classic as well as sensu go 5.16+ to use v4

Reply:
Change added